cmd/cmd_controller.js

```diff
@@ -59,19 +59,19 @@ class EmbarkController {
 
     const webServerConfig = {};
 
-    if (options.runWebserver !== null) {
+    if (options.runWebserver !== null && options.runWebserver !== undefined) {
       webServerConfig.enabled = options.runWebserver;
     }
 
-    if (options.serverHost !== null) {
+    if (options.serverHost !== null && options.serverHost !== undefined) {
       webServerConfig.host = options.serverHost;
     }
 
-    if (options.serverPort !== null) {
+    if (options.serverPort !== null && options.serverPort !== undefined) {
       webServerConfig.port = options.serverPort;
     }
 
-    if (options.openBrowser !== null) {
+    if (options.openBrowser !== null && options.openBrowser !== undefined) {
       webServerConfig.openBrowser = options.openBrowser;
     }
 
```

Comment:
Notice that we could probably also do a positivity check a la

```
if (options.serverPort) {

}
```

etc. That however won't work well for options like `runWebserver` and `openBrowser`, which could be `false` and should still be set.

Reply:
Can we review and compare the behavior prior to the regression? I wrote it the way I did to avoid a situation where *missing* cli options were effectively overriding options specified in a DApp's `config/` files.

Reply:
For reference:

https://github.com/embark-framework/embark/commit/2b5268dc846bec482028bee41cb1073739926b30

https://github.com/embark-framework/embark/commit/638fb4ee27352626321e2e807cbc10a02de62fdc

https://github.com/embark-framework/embark/commit/a966ecb1310ceaf3b20d0324123d1fd3663d38e1

https://github.com/embark-framework/embark/commit/64027a5ee429c081c5d51fda7ea14ed1edf8d761

At one point, the `nullify()` helper stop being used, I think the parity support PR changed that.

Reply:
@michaelsbradleyjr with this patch it should work as before. It's a little tricky to wrap ones head around this due to positive negativity, but basically before the this patch, if an options was either `null` or `undefined` the options has **not** been set.

With this patch, this behavior is recovered. 

Do you need anything else? Let me know if so!

Reply:
I agree it does seem to restore the original behavior. In which case, this helper can be removed, as it's not being used:

https://github.com/embark-framework/embark/blob/fix/embark-run-command/cmd/cmd.js#L169